haven/citadel/cit_miscreants.dm

```diff
@@ -0,0 +1,66 @@
+/datum/controller/subsystem/ticker/proc/generate_miscreant_objectives(var/datum/mind/crewMind)
+	if(!GLOB.miscreants_allowed)
+		return
+	if(!crewMind)
+		return
+	if(!crewMind.current || !crewMind.objectives || crewMind.special_role)
+		return
+	if(!crewMind.assigned_role)
+		return
+	if(!(ROLE_MISCREANT in crewMind.current.client.prefs.be_special))
+		return
+	if(jobban_isbanned(crewMind, "Syndicate"))
+		return
+	var/list/objectiveTypes = miscreantobjlist
+	if(!objectiveTypes.len)
+		return
+	var/selectedType = pick(objectiveTypes)
+	var/datum/objective/miscreant/newObjective = new selectedType
+	if(!newObjective)
+		return
+	newObjective.owner = crewMind
+	crewMind.objectives += newObjective
+	crewMind.special_role = "miscreant"
+	to_chat(crewMind, "<B><font size=3 color=red>You are a Miscreant.</font></B>")
+	to_chat(crewMind, "<B>Pursuing your objective is entirely optional, as the completion of your objective is unable to be tracked. Performing traitorous acts not directly related to your objective may result in permanent termination of your employment.</B>")
+	to_chat(crewMind, "<B>Your objective:</B> [newObjective.explanation_text]")
+
+/datum/objective/miscreant
+	explanation_text = "Something broke. Horribly. Dear god, im so sorry. Yell about this in the development discussion channel of citadels discord."
```

Comment:
Ditto.

Reply:
Uh... Is this a common issue?

Reply:
Surprisingly, no. The only time I've really seen the message appear on live was when admins were involved. Keeping the message there leaves a failsafe in the event that something breaks the code, like upstream mirrors, or other contributors.